Sources/Hyperspace/Service/Transport/TransportService.swift

```diff
@@ -14,6 +14,7 @@ public class TransportService {
     // MARK: - Properties
     
     let session: TransportSession
+    private let queue = DispatchQueue(label: "Hyperspace.tasks.sync")
```

Comment:
Went back and checked and `NetworkActivityController` does have it's own queue, so regardless of the solutions for the below, I think we should match it's label `com.bottlerocketstudios.hyperspace.tasks` (or something).